### DISCUSSION TITLE
Static content translations, inside a component or nah?

### DISCUSSION THREAD

**Discussion by zeroedin** [RedHat-UX/red-hat-design-system#583] [General]
`<rh-secondary-nav>` contains static content of `Menu` for the mobile menu button.  Should the translations for the "Menu" content be housed inside the component or should there be an API to change it and allow the implementor to decide?  Maybe both?  The auto-translation checking `lang` defaulting to English if not present and an attribute to override it completely? 

Issue: https://github.com/RedHat-UX/red-hat-design-system/issues/584

**Comment by bennypowers** [comment]
This could be a slot?

**Reply by zeroedin** [reply]
Yes, and that's how I implemented it.  #591
